### DISCUSSION TITLE
Checkbox isue

### DISCUSSION THREAD

**Discussion by mortezaahmadiaf** [primefaces/community#1790] [PrimeReact] [answered]
i import Check box like 
import { Checkbox, CheckboxProps } from 'primereact/checkbox'
and user it like 
 <Checkbox
        id={"id"}
        checked={true}
      />
but in view checkbox not Checked 
please help me to fix it

**Comment by melloware** [comment, marked as answer]
Duplicate of already answered: https://github.com/primefaces/primereact/issues/6321
